test/Interop/Cxx/foreign-reference/Inputs/module.modulemap

```diff
@@ -49,3 +49,7 @@ module DerivedFieldGetterReturnsOwnedFRT {
   requires cplusplus
 }
 
+module FunctionsAndMethodsReturningFRT {
+  header "cxx-functions-and-methods-returning-frt.h"
+  requires cplusplus
```

Comment:
That's technically possible with `-experimental-c-foreign-reference-types` flag. However, the only test that we have (`Interop/C/struct/foreign-reference.swift`) is disabled since 2022, so it's unlikely that it's going to work now.

Reply:
hmm if FRT in C is not a well supported feature now, should I avoid adding a C language mode test ?

Reply:
Yeah, I think we can skip that test.